charts/deployment-operator/templates/deployment.yaml

```diff
@@ -175,6 +180,11 @@ spec:
             defaultMode: 0444
             name: {{ include "deployment-operator.fullname" . }}
         {{- end }}
+        {{- range $cert := .Values.certs }}
+        - name: {{ $cert.name }}
+          configMap:
+            name: {{ $cert.name }}
```

Comment:
I'm pretty sure this needs to be `{{ include "deployment-operator.fullname" $ }}-{{ $cert.name }}` right?